packages/template/base/src/BaseTemplate.ts

```diff
@@ -47,12 +47,17 @@ export class BaseTemplate implements ForgeTemplate {
           }
           const srcFiles = ['index.css', 'index.js', 'index.html', 'preload.js'];
 
-          for (const file of rootFiles) {
-            await this.copy(path.resolve(tmplDir, file), path.resolve(directory, file.replace(/^_/, '.')));
-          }
-          for (const file of srcFiles) {
-            await this.copy(path.resolve(tmplDir, file), path.resolve(directory, 'src', file));
-          }
+          await Promise.all(
+            rootFiles.map(async (file) => {
+              await this.copy(path.resolve(tmplDir, file), path.resolve(directory, file.replace(/^_/, '.')));
```

Comment:
likewise should I remove the await on line 52 and 58?

Reply:
Yes